client/src/pages/private/ParticipantTable.js

```diff
@@ -173,27 +172,33 @@ export default () => {
   };
 
   const handleFsaFilter = (value) => {
-    setPagination(prev => ({
-      ...prev,
+    if(fsaText?.trim() === value?.trim()) return;
+
+    setPagination(oldPagination => ({
+      ...oldPagination,
       currentPage: 0,
     }));
-    setFsaFilter(value);
+    setFsaFilter(value.trim());
   };
 
   const handleLastNameFilter = (value) => {
-    setPagination(prev => ({
-      ...prev,
+    if(lastNameText?.trim() === value?.trim()) return;
+    
+    setPagination(oldPagination => ({
```

Comment:
I'm doing a bad job reviewing this... I'm so sorry, these `old____` variables were recently changed to `prev` for consistency, I think this is an unintentional change

Reply:
All good. I'll make the change.